README.md

```diff
@@ -1,64 +1,46 @@
-=====
+Perseverance Modelica
+=====================
 
-This is a model of the Perseverance rover.
+This repository contains Modelon experiments for NASA's Perseverance and Ingenuity platforms.
 
-![driving over uneven ground](Perseverance/Resources/images/driveUneven.png)
+# Experiments
 
-![driving over uneven ground](Perseverance/Resources/images/driveUneven.gif)
+* [Perseverance](Perseverance/README.md); model of the Perseverance rover.
+* [Ingenuity](Ingenuity/README.md); model of the Ingenuity helicopter.
+* [Example Models](ExampleModels/README.md); experiments showing how both the Perseverance and Ingenuity models can be simulated together.
 
-Prerequisites
--------
-The model package is based on **Vehicle Dynamics Library** and **Electrification Library** from Modelon. All prerequisites are included in an installation of **Modelon Impact Pro**. The models can also be used in Dymola with separate installations of the required libraries, though the visualizers (glb files) are not supported.
+# Compatibility with Modelon Impact Versions
 
-Quick start
---------
+| Github Branch                                                                              | Modelon Impact Version |
+|--------------------------------------------------------------------------------------------|------------------------|
+| [vdl-4.1](https://github.com/Open-MBEE/perseverance-modelica/tree/vdl-4.1)                 | Impact 2022.2 (1.8.)   |
+| [vdl-4-2-upgrade](https://github.com/Open-MBEE/perseverance-modelica/tree/vdl-4-2-upgrade) | Impact 2023.1 (1.10.)  |
+| [vdl-4-3-upgrade](https://github.com/Open-MBEE/perseverance-modelica/tree/vdl-4-3-upgrade) | Impact 2023.2 (2.9.*)  |
 
-In Perseverance.Experiments there are six example experiments that can be simulated out of the box:
- - **FlatPad** - Standing still on flat ground
- - **DrivingFlat** - Driving forward on flat ground at a constant speed
- - **DrivingCleats** - Driving over a rectangular cleat at a constant speed
- - **Spin** - Spin in place by driving wheels on each side in opposite directions and adding the appropriate steering angles to the corner wheels
- - **DrivingUneven** - Driving forward over uneven ground generated using the Spirit_landing_site 3D model
- - **DrivingUneven2** - Driving forward over uneven ground generated using the M2020 ZCAM -- Hogwallow Flats, Sol 461 3D model
+# Getting Started
 
-Acknowledgements
------------
-The visualization elements are extracted from https://github.com/nasa/NASA-3D-Resources/tree/master/3D%20Models/perseverance-GLB. This is also where the main geometrical parameters are gathered.
+Once logged into Modelon Impact Pro...
 
-The visualizer Perseverance/Resources/Spirit_landing_site.glb  is extracted from https://github.com/nasa/NASA-3D-Resources/tree/master/3D%20Models/Spirit%20Landing%20Site. This is also used to generate the VDL ground model data used in Perseverance.Experiments.DrivingUneven.
+1. Select **Repository Management** from the Apps menu.
```

Comment:
This part is specific to an older version of Modelon Impact. We will update it later.